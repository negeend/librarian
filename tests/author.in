AUTHOR Douglas Adams
author Douglas Adams
ADD COLLECTION books3.csv
AUTHOR Douglas Adams
AUTHOR douglas ADAMS
author Jamie Lee Curtis
author Roald Dahl
AUTHOR INFO1113
exit
