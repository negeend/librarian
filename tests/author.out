user: No books in system.

user: No books in system.

user: 22 books successfully added.

user: The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)

user: The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)

user: Halloween (Jamie Lee Curtis)

user: No books by Roald Dahl.

user: Best Unit Ever (INFO1113)

user: Ending Library process.
